deeppavlov/models/kbqa/utils.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple, List
+import re
+
+def extract_year(question_tokens: List[str], question: str) -> str:
+    year = ""
+    fnd = re.search(r'.*\d/\d/(\d{4}).*', question)
```

Comment:
Here and below you are not parsing date fith two-digit month or day. See PR #1192 suggestion